### PR TITLE
Add new aws.elb properties + fix createdTime consistency

### PR DESCRIPTION
**Pull request by tas50**
New properties:

- createdTime
- vpcID

Try to make create time properties a bit more consistent. The Amazon API is not consistent, which is a bummer.

### CODE REVIEW

**Comment by vjeffrey** [comment]
im seeing some errors, ill try to take a look tonight:
```
   }
    ]
    scheme: "internet-facing"
    createdTime: cannot convert primitive with NO type information
    arn: "arn:aws:elasticloadbalancing:us-east-2:921877552404:loadbalancer/app/vj-test/87f426f00d031f77"
    dnsName: "vj-test-785441053.us-east-2.elb.amazonaws.com"
    vpcID: cannot convert primitive with NO type information
    name: "vj-test"
  }
```

File: providers/aws/resources/aws_elb.go [hunk 1]
```diff
@@ -73,9 +73,11 @@ func (a *mqlAwsElb) getClassicLoadBalancers(conn *connection.AwsConnection) []*j
...
 							"arn":                  llx.StringData(fmt.Sprintf(elbv1LbArnPattern, regionVal, conn.AccountId(), convert.ToString(lb.LoadBalancerName))),
 							"listenerDescriptions": llx.AnyData(jsonListeners),
-							"dnsName":              llx.StringData(convert.ToString(lb.DNSName)),
-							"name":                 llx.StringData(convert.ToString(lb.LoadBalancerName)),
-							"scheme":               llx.StringData(convert.ToString(lb.Scheme)),
+							"dnsName":              llx.StringDataPtr(lb.DNSName),
+							"name":                 llx.StringDataPtr(lb.LoadBalancerName),
+							"scheme":               llx.StringDataPtr(lb.Scheme),
+							"vpcID":                llx.StringDataPtr(lb.VPCId),
+							"createdTime":          llx.TimeData(toTime(lb.CreatedTime)),
```
Comment (preslavgerchev) [inline providers/aws/resources/aws_elb.go]: nit: I think if this ever returns `nil` (which it probably will not for the *creation*), the time here will show up as `1001-01-01` as `toTime` initializes an empty time struct. Better to use `llx.TimeDataPtr` directly which will handle nil accordingly by not returning an empty value

Comment (tas50) [reply providers/aws/resources/aws_elb.go]: Thanks for the explaination and the fix @preslavgerchev

**Review by preslavgerchev — APPROVED** [review]

**Comment by preslavgerchev** [comment]
merging this, despite the failed build as that's only the CLA assistant.

**Pull request merged**